packages/paste-website/src/pages/inclusive-design/index.mdx

```diff
@@ -0,0 +1,419 @@
+        <ListItem>Make UI controls look like controls; give them strong perceived affordance</ListItem>
+        <ListItem>Avoid pure white or pure black shades</ListItem>
+        <ListItem>Indicate swipe gesture support clearly, and provide simple tap-based alternatives</ListItem>
+        <ListItem>
+          Avoid time constraints where possible; provide a clear warning and option to extend where not possible
+        </ListItem>
+        <ListItem>Ensure primary calls to action are easy to recognize and reach</ListItem>
+        <ListItem>Use well-established, therefore recognizable, icons and symbols</ListItem>
+        <ListItem>Don't make users perform actions to reveal content unless completely necessary</ListItem>
+        <ListItem>If content is meant to be hidden, ensure it is properly hidden to all users</ListItem>
+        <ListItem>Inform the user when there are important changes to the application state</ListItem>
+      </UnorderedList>
+    </DisclosureContent>
+  </Disclosure>
+  <Disclosure>
+    <DisclosureHeading as="h3" variant="heading30">
+      Other considerations
+    </DisclosureHeading>
+    <DisclosureContent>
+      <UnorderedList>
+        <ListItem>Use the same design patterns to solve the same problems</ListItem>
+        <ListItem>Label and describe the same things with the same terminology</ListItem>
+        <ListItem>Do not trigger "infinite scroll" by default; provide buttons to load more items</ListItem>
+      </UnorderedList>
+    </DisclosureContent>
+  </Disclosure>
+</Stack>
+
+## Considerations for UX and front-end engineers
+
+While the tips below are geared towards developers, it is recommended that designers understand these concepts and have conversations about inclusive design with their engineering partners.
+
+<Stack orientation="vertical" spacing="space30">
+  <Disclosure>
+    <DisclosureHeading as="h3" variant="heading30">
+      Performance considerations
+    </DisclosureHeading>
+    <DisclosureContent>
+      <UnorderedList>
+        <ListItem>
+          Minify CSS and JS, and remove unused/redundant code{' '}
+          <Anchor showExternal href="https://developers.google.com/speed/docs/insights/MinifyResources">(Read more about why this matters)</Anchor>
+        </ListItem>
+        <ListItem>
+          Compress raster images{' '}
+          <Anchor showExternal href="https://www.html5rocks.com/en/tutorials/speed/img-compression/">(Read more about why this matters)</Anchor>
+        </ListItem>
+        <ListItem>
+          Optimize SVG path data{' '}
+          <Anchor showExternal href="https://web-design-weekly.com/2014/10/22/optimizing-svg-web/">(Read more about why this matters)</Anchor>
+        </ListItem>
+        <ListItem>
+          Make sure styles and scripts are not render blocking{' '}
+          <Anchor showExternal href="https://csabapalfi.github.io/eliminate-render-blocking/">(Read more about why this matters)</Anchor>
```

Comment:
Broken link. I suggest https://developers.google.com/web/fundamentals/performance/critical-rendering-path/render-blocking-css and https://developers.google.com/speed/docs/insights/BlockingJS